src/resources/vue/scss/global.scss

```diff
@@ -1,13 +1,19 @@
 @import url('https://fonts.googleapis.com/css2?family=Outfit:wght@100;200;300;400;500;600;700;800;900&display=swap');
 @import './variables.scss';
 
-html, body {
+html,
+body {
     font-family: $font-main;
-    background-color: $color-dark-blue-100;
+    //background-color: $color-dark-blue-100;
```

Comment:
why is this commented out? I think this needs to be used

Reply:
fixed